portal/server/routes/middleware/rateLimit/index.js

```diff
@@ -0,0 +1,32 @@
+const limitter = require('express-rate-limit');
+
+const { RATE_LIMIT_THRESHOLD } = process.env;
+
+/**
+ * Global rate-limitter middleware
+ * @param {Object} req Request object
+ * @param {Object} res Response object
+ */
+const rateLimit = (req, res) => {
+  if (!RATE_LIMIT_THRESHOLD) {
+    console.error('Invalid rate limit threshold value %d', RATE_LIMIT_THRESHOLD);
+    return res.redirect('/not-found');
+  }
+
+  return limitter({
+    // 1 minutes
+    windowMs: 1 * 60 * 1000,
+    // Threshold limit, x requests per windowMs (60 seconds)
+    max: RATE_LIMIT_THRESHOLD,
```

Comment:
Were we not using a unique one for each service or just dtfs shared?

Reply:
The name will be common, but values for each `.env` inside their respective micro-services can be different.